tests/test_raster_pipe.py

```diff
@@ -48,14 +48,13 @@ def test_create_tiles_subset():
         ), mock.patch(
             "gfw_pixetl.utils.upload_geometries.upload_geojsons", return_value=None
         ):
-            (
-                tiles,
-                skipped_tiles,
-                failed_tiles,
-            ) = PIPE.create_tiles(overwrite=True)
+            (tiles, skipped_tiles, failed_tiles, existing_tiles) = PIPE.create_tiles(
+                overwrite=True
+            )
             assert len(tiles) == 1
             assert len(skipped_tiles) == 3
             assert len(failed_tiles) == 0
+            assert len(existing_tiles) == 0
```

Comment:
this or a similar test would be a good place to test if existing tiles are correctly identified and carries through the entire pipe.

Reply:
Done.